net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/proxy.conf

```diff
@@ -0,0 +1,128 @@
+{% if helpers.exists('OPNsense.freeradius.general.fallback') and OPNsense.freeradius.general.fallback == '1' %}
```

Comment:
What is this? Did you test this in production in all possible ways?

Reply:
I'm sorry copying the files to send a clean pull request I haven't copyed a file, now I check and add the general file edit to enable fallback. I'm using in production with 12 schools, all with different configurations.

Reply:
Tomorrow I'll test another time the plugin in production taking attention to fallback options.

Reply:
Hi @mimugmail I have tested and pushed to production to schools that I follow.
There was another problem in all possible configurations and I fixed it, tested and pushed now.